todo.js

```diff
@@ -0,0 +1,110 @@
+let todos = [
```

Comment:
You'll want to make sure your updating this branch from `main` after your JS rendering branch gets merged.